tests/utils/dockercli/volumelifecycle.go

```diff
@@ -171,6 +183,17 @@ func StopContainer(ip, containerName string) (string, error) {
 	return ssh.InvokeCommand(ip, dockercli.StopContainer+containerName)
 }
 
+// AttachVolume - attach volume to container on given host
+func ExecContainer(ip, volName, containerName string) (string, error) {
+	log.Printf("Attaching volume [%s] on VM [%s]\n", volName, ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.RunContainer+" -d --rm -v "+volName+
+			":/vol1 --name "+containerName+ dockercli.TestContainer)
+	if err != nil {
+		return out, err
+	}
+	return ssh.InvokeCommand(ip, dockercli.RemoveContainer + containerName)
```

Comment:
The --rm flag doesn't remove the container till the container exits. And the container won't exit till the command being run exits. Doing a "tail -f on /dev/null" never exits. I'm not changing that in this PR.